src/BuildContainer/BuildContainer.php

```diff
@@ -191,6 +191,23 @@ protected function get_CellMatrix() {
         }
       }
     }
+    
+    $cellAttributesIndexed = [];
+    foreach ($this->CellAttributes as $rowName => $rowCellAttributes) {
+    	if ($this->rows->nameIsLeaf($rowName)) {
+    		$iRow = $this->rows->subtreeIndex($rowName);
+    		$rowCellAttributesIndexed = [];
+    		foreach ($rowCellAttributes as $colName => $attributes) {
+    			if ($this->columns->nameIsLeaf($colName)) {
+    				$iCol = $this->columns->subtreeIndex($colName);
+    				$rowCellAttributesIndexed[$iCol] = $attributes;
+    			}
+    		}
+    		$cellAttributesIndexed[$iRow] = $rowCellAttributesIndexed;
+    	}
+    }
+    
+    $matrix->setCellAttributes($cellAttributesIndexed);
```

Comment:
Indentation in this PR does not align with the rest of the package.
I know the rest of the php world uses 4 spaces, but we need to be consistent within the package first.
(the 2 spaces are because I work with Drupal most of the time)